src/util/FactorUtil.js

```diff
@@ -129,6 +129,12 @@ function (Okta, TimeUtil) {
       description: Okta.loc('factor.customFactor.description', 'login'),
       iconClassName: 'mfa-custom-factor',
       sortOrder: 16
+    },
+    'CUSTOM_HOTP': {
+      label: '',
+      description: '',
+      iconClassName: 'mfa-custom-factor',
+      sortOrder: 17
```

Comment:
this should be a higher sort order I assume, since this is similar to other OTP methods we have. Should probably be right after Google Auth or if not after RSA, but don't think this should be at the bottom of the list since we know the strength of this factor